### ISSUE TITLE
schema files location

### ISSUE THREAD

**Issue by joschirr** [openaire/guidelines-cris-managers#51]
Schema files could be stored on the OpenAIRE server and referenced from

https://www.openaire.eu/schema/cris/[version number]/

**Comment by jdvorak001** [comment]
Yes, it will be very good to have an official location. To be implemented in the upcoming 1.1.1 release.

It will need to be propagated in the sample files, too.

**Comment by jdvorak001** [comment]
Actually, do we want version number 1.1, or finer and separate 1.1.0, 1.1.1, ... ?

**Comment by joschirr** [comment]
the version number scheme should be consistent with the release of the guidelines.
According to the current release 1.1.0 the schema files are available under
https://www.openaire.eu/schema/cris/1.1.0/

**Comment by ACz-UniBi** [comment]
The schema version is available at
v1.1.0 : https://www.openaire.eu/schema/cris/1.1.0/
v1.1.1:  https://www.openaire.eu/schema/cris/1.1.1/

I would prefer also the latest v1.1 at:
https://www.openaire.eu/schema/cris/1.1/    that's linked to v1.1.1

**Comment by jdvorak001** [comment]
@ACz-UniBi : Yes, that's how it's currently implemented and probably a good convention to keep for future releases, too.
